components/list/src/provider/serde_dfa.rs

```diff
@@ -128,23 +128,33 @@ impl<'data> SerdeDFA<'data> {
     /// Creates a `SerdeDFA` from a regex.
```

Comment:
Thought: `DFA::from_bytes` didn't change from 0.2 to 0.4 ?

Reply:
define change?

Reply:
I was just expecting to see changes in how we deserialize the bytes but I only saw changes in how we build them.